### PR TITLE
[Ready] His Grace Ascension RETURNS at 25 victim requirement

### PR DESCRIPTION
**Pull request by Militaires**
Sprites by WJohnston

**Tl;dr when his grace eats 25 people he becomes golden (+bonus to his force) and his bloodthirst is locked at its current level, so if it happens to reach 25 people while enraged, it will stay that way and never go back to sleep, if it ascends at another bloodthirst level you're essentially blessed with not having to worry about its hunger anymore; that is if you can reach 25 people without being eaten yourself, this is very, VERY difficult.**

This time the ascension requirement is 25 people. Reaching this number is almost impossible, because the more people it has in it the hungrier it becomes, so if you're able to feed 25 people in that VERY short time you have until it turns against you, you deserve your reward. 

Last time, this was reverted because it makes his grace a "permanent" hazard on the station, what was NOT noted however is the fact that reaching 25 people is extremely difficult anyways, and this:

Another method, is to literally not walk near the GOLDEN BLOODY TOOLBOX and it can't hurt you, that's literally it.

Another argument I heard was that this is bad because not all of the crew can destroy the toolbox almost instantly like literally anyone with a gun can do because the ENTIRE crew needs to be able to shoot it dead (which it already can, just order or steal a laser rifle), you might think that this argument is absolutely brainlet-tier, in which case you're just absolutely right and nothing else.

:cl: Basilman, Sprites by WJohnston
add: His Grace ascension is back, feed Him 25 people and you will unlock His full potential.
/:cl:

Funny thing is,  this was reverted so fast that I don't think it even happened once ingame yet, nor did anyone give a comment about it.

### CODE REVIEW

**Comment by Wilchenx** [comment]
i managed to do it once

image

**Comment by ExcessiveUseOfCobblestone** [comment]
Is 25 based off the nuke killstreak in cod by chance?

**Comment by Kagus** [comment]
I mean, you can also just mutadone-smoke a bunch of monkeys and devour their corpses too, but this is neat.

Can you still devour corpses post-ascension? Is there any reason to?

**Comment by Militaires** [comment]
well if you order a bunch of monkey boxes, go to genetics, prepare mutadone, prepare smoke, smoke them, and then feed the box the 25 people without nothing getting in your way yeah you deserve it

@ExcessiveUseOfCobblestone idk i never played cod

i just made it 25 so it's even harder to reach the thing

**Comment by Kagus** [comment]
Or just feed a biogenerator watermelons and get a chem dispenser. Dispenser's the real bottleneck here, so you'll need to either try and finagle one out of chem, grab a board off of Sci or just re-steal the one that botany inevitably steals.

Unless you can somehow convince someone that it's your holy duty to spread carpeting through the station, and the daft bastards just give you a dispenser to do it with...

File: code/game/objects/items/his_grace.dm [hunk 1]
```diff
@@ -229,3 +234,17 @@
...
+/obj/item/his_grace/proc/ascend()
+	if(ascended)
+		return
+	var/mob/living/carbon/human/master = loc
+	force_bonus += ascend_bonus
+	desc = "A legendary toolbox and a distant artifact from The Age of Three Powers. On its three latches engraved are the words \"The Sun\", \"The Moon\", and \"The Stars\". The entire toolbox has the words \"The World\" engraved into its sides."
+	icon_state = "his_grace_ascended"
+	item_state = "toolbox_gold"
+	ascended = TRUE
+	playsound(master, 'sound/effects/his_grace_ascend.ogg', 100)
```
Comment (Cyberboss) [inline code/game/objects/items/his_grace.dm]: master => src

File: code/game/objects/items/his_grace.dm [hunk 2]
```diff
@@ -229,3 +234,17 @@
...
+	if(istype(master))
+		master.visible_message("<span class='his_grace big bold'>Gods will be watching.</span>", "<span class='his_grace big bold'>God will be watching.</span>")
```
Comment (Cyberboss) [inline code/game/objects/items/his_grace.dm]: second parameter isn't necessary
